test/integration_tests.py

```diff
@@ -1,10 +1,26 @@
 import asyncio
 import asynqp
 import socket
+import urllib.request
+import base64
 import contexts
+import json
 from .util import testing_exception_handler
 
 
+def _deep_compare(a, b):
```

Comment:
This looks like it does the same thing as `==` does by default. Am I missing something?